### ISSUE TITLE
Stomp alerter code (stomp.py) slowing elastalert startup time significantly 

### ISSUE THREAD

**Issue by beezz** [Yelp/elastalert#1791]
Resolution of all local hostnames in [stomp.py utils](https://github.com/jasonrbriggs/stomp.py/blob/master/stomp/utils.py#L20) causes slow elastalert start.

With stomp alerter:
```
» time python -c "from elastalert import elastalert"
python -c "from elastalert import elastalert"  0.42s user 0.06s system 1% cpu 40.895 total
```

Without stomp alerter:
```
» time python -c "from elastalert import elastalert"
python -c "from elastalert import elastalert"  0.32s user 0.05s system 98% cpu 0.382 total
```

Solution might be to do late import on stomp alerter class instatiation. Let me know what do you think and I can prepare PR. For now I'll have fork with Stomp alerter and stomp.py as dependency removed.

**Comment by Qmando** [comment]
Sounds like you've got a bad DNS configuration. I'm guessing you've got a large number of DNS search domains or the resolver is freezing on some upstream DNS server. 

If you attempt to resolve other, non-existent domains, does it always return immediately?

**Comment by beezz** [comment]
@Qmando Indeed the extremity (40s) was my DNS config. Anyway, doing blocking calls to like stomp.py is doing during import time does not sound like a good idea. Especially when running in [lambda](https://github.com/beezz/elastalert-lambda) where every millisecond counts.

**Comment by Qmando** [comment]
Yeah, I agree it's a bad thing to do at import time. Doesn't look like this will get fixed upstream either. https://github.com/jasonrbriggs/stomp.py/issues/186

As I said, I'd be happy to move the imports around, so that they are only used if the alerter is being used, like you suggested.
